helpers/recipesDB.js

```diff
@@ -0,0 +1,23 @@
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+
+module.exports = {
+    getDishes: function(){
+        return db('dishes')
+    },
+    addDish: function(dish){
+        return db('dishes').insert(dish)
+    },
+    getDish: function(id){
+        return db('dishes').where(id)
+    },
+    getRecipes: function(){
+        return db('recipes');
```

Comment:
That one is incorrect. You need to perform some joins on the table to also get the dishes that it belongs to.